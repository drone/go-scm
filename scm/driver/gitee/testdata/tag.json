{
  "name": "1.0",
  "message": "1.0",
  "commit": {
    "sha": "5e7876efb3468ff679410b82a72f7c002382d41e",
    "date": "2021-03-24T11:29:35+08:00"
  }
}