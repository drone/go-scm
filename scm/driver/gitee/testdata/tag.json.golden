{
  "name": "1.0",
  "Path": "refs/tags/1.0",
  "Sha": "5e7876efb3468ff679410b82a72f7c002382d41e"
}